stream/tests/test-stream.js

```diff
@@ -152,12 +152,10 @@ o.spec("stream", function() {
 			o(b()()).equals(undefined)
 		})
 		o("combine can halt", function() {
-			var count = 0
 			var a = Stream(1)
 			var b = Stream.combine(function() {
 				return Stream.HALT
 			}, [a])["fantasy-land/map"](function() {
-				count++
```

Comment:
I will add the assertions. But i removed these lines and they were not removed automatically. Eslint fix safe when it comes to auto fixing.